packages/ui/src/data/filters/utils.ts

```diff
@@ -0,0 +1,272 @@
+import qs from 'query-string';
+import get from 'lodash/get';
+import { isEmpty } from 'lodash';
+import { IFilter, IFilterCount, IFilterGroup, IFilterRange, VisualType } from '../../components/filters/types';
+import {
+    ISyntheticSqon,
+    IValueContent,
+    IValueFilter,
+    TSqonContent,
+    TSqonGroupOp,
+    TSyntheticSqonContent,
+} from '../sqon/types';
+import xss, { IFilterXSSOptions } from 'xss';
+import { isFieldOperator, isNotReference, isReference } from '../sqon/utils';
+import { BooleanOperators, FieldOperators } from '../sqon/operators';
+
+const xssConfig: IFilterXSSOptions = {
```

Comment:
why not using the library with default configs? We need tests for injection I think.

Reply:
I only reused what was used in CQDG.  Not sure what library could be used here.